### PR TITLE
MSS section updates

### LINKED ISSUE #5237: Minimum Security Standard updates
### Todo
- [x] Hide MSS badge from hero banner
- [x] Re-style the criteria section (only use font styles that are already in use on production)
- [ ] Add accordion (default state is closed) - will be covered in #5257
- [x] Show privacy ding if answer is `No` - [icon](https://drive.google.com/drive/folders/1kPMOO9YoMybLN-MqziLUsmGE2EQ3hAVp?usp=sharing)
- [x] Move section down the page 

### Closed state desktop
image

### Open state desktop
image

### Closed state mobile
image

### Open state mobile
image

---

###  Design -> Dev Handoff Checklist

- [x] Design has been finalized
- [x] Link to original design ticket #5152
- [X] Link to related ticket #5257
- [x] Link to [Sketch file](https://share.goabstract.com/584e7002-4d07-4ce5-aafe-dc825e076a27?sha=f0b8ca1b588425d1ef83c57a9f4126c8ff97648e)
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included
- [x] Assets are included - [icon](https://drive.google.com/drive/folders/1kPMOO9YoMybLN-MqziLUsmGE2EQ3hAVp?usp=sharing)
- [x] Hover state for elements are designed - no hover state for the accordion / link: use what is on production
- [x] Other states for elements are designed (e.g., error state for form) - n/a

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5237 

Code reviewer:
I fixed a bunch indentation issues so it's best to view code difference **without** whitespace changes.

----

Please only review the "Security" section. Fixes to other sections will be handled in other tickets.

image

- has ding: https://foundation-s-issue-5237-qfk8ve.herokuapp.com/en/privacynotincluded/products/percy-cypress/
- no ding: https://foundation-s-issue-5237-qfk8ve.herokuapp.com/en/privacynotincluded/products/short-of/

### CODE REVIEW

**Review by kristinashu — APPROVED** [review]
Looks good and the ding is showing / not showing as expected!

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 1]
```diff
@@ -81,12 +80,12 @@ <h2 class="h3-heading">{% trans "What could happen if something goes wrong" %}</
...
 
-        <section>
-          <h2 class="h2-heading">{% trans "Privacy" %}</h2>
+        <section class="criterion-group">
+          <h2 class="h3-heading">{% trans "Privacy" %}</h2>
 
           {% if product.product_type == 'general' %}
           <section>
-            <h3 class="h3-heading">{% trans "Can it snoop on me?" %}</h3>
+            <h3 class="pni-product-smaller-body-bold">{% trans "Can it snoop on me?" %}</h3>
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html:88]: ❤️

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 2]
```diff
@@ -229,81 +228,41 @@ <h3 class="h3-heading">{% trans "Can I control it?" %}</h3>
...
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.security_updates        help=product.security_updates_helptext        label=security_updates          %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.manage_vulnerabilities  help=product.manage_vulnerabilities_helptext  label=manages_vulnerabilities   %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.privacy_policy          help=product.privacy_policy_helptext          label=privacy_policy            %}
-          </div>
-        </div>
-      </div>
+      <section class="minimum-security criterion-group">
+        <h2 class="h3-heading">{% trans "Security" %}</h2>
+        {% trans "Does this product meet our <a href='https://foundation.mozilla.org/privacynotincluded/about/meets-minimum-security-standards'>Minimum Security Standards</a>?" as minimum_security_standards %}
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html:233]: This one is... tricky. @TheoChevalier do you know if we can blocktrans here, to keep that `url` from being part of the translation string itself?

**Review by Pomax — CHANGES_REQUESTED** [review]
Whoo! Almost R+ but that one translation looks like it's going to be a problem, so I flagged Theo for help

File: network-api/networkapi/buyersguide/templates/product_page.html (hunk 2 above)
Comment (TheoChevalier) [reply network-api/networkapi/buyersguide/templates/product_page.html:233]: Tricky indeed, but turns out it’s possible using `asvar`!
Filed a PR to be merged with this one here https://github.com/mozilla/foundation.mozilla.org/pull/5347

Comment (mmmavis) [reply network-api/networkapi/buyersguide/templates/product_page.html]: thanks!

**Comment by mmmavis** [comment]
@Pomax this PR now has Theo's i10n fixes!

**Review by Pomax — COMMENTED** [review]
yay!

**Review by Pomax — APPROVED** [review]

**Pull request merged**